bftengine/src/simplestatetransfer/SimpleStateTran.cpp

```diff
@@ -61,8 +61,8 @@ class SimpleStateTran : public ISimpleInMemoryStateTransfer {
                              uint16_t sizeOfDigestBuffer,
                              uint64_t& outBlockId,
                              char* outStateDigest,
-                             char* outFullStateDigest) override;
-
+                             char* outResPagesDigest,
```

Comment:
Same comment for all the functions taking char* digest to convert the type to Digest& if that's better.

Reply:
See my answer above, I think it was intentionally made to be char*, although not really used for now.